examples/for-tests/src/App.js

```diff
@@ -124,7 +124,21 @@ const formFields = [
     },
 ];
 
-let recipeList = [Session.init()];
+let recipeList = [
+    Session.init({
+        preAPIHook: (ctx) => {
+            // FIXME: Workaround the issue events firing differently on test runs
```

Comment:
This is tracked in https://github.com/supertokens/supertokens-auth-react/issues/267